monitor/common.go

```diff
@@ -148,3 +148,15 @@ func (m *Monitor) setUnbonded() {
 	defer m.lock.Unlock()
 	m.bonded = false
 }
+
+func (m *Monitor) isBootstrapped() bool {
+	m.lock.RLock()
```

Comment:
why lock is necessary?

Reply:
`processPullerQueue` reads and `monitorDPoSValidatorChange` writes, they are two threads, so technically a race condition is possible. However, we only set once in the bootstrap node at the beginning of the sidechain setup, so no lock could be fine and this might be an overkill

Reply:
let's keep the lock, as even `bootstrapped` is already true, it still needs to be accessed for each `AddValidator` event.